open-metadata-implementation/common-services/ffdc-services/src/main/java/org/odpi/openmetadata/commonservices/ffdc/rest/FFDCResponse.java

```diff
@@ -0,0 +1,156 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.commonservices.ffdc.rest;
+
+import java.util.Map;
+
+public interface FFDCResponse {
+
+    /**
+     * Return the HTTP Code to use if forwarding response to HTTP client.
+     *
+     * @return integer HTTP status code
+     */
+    int getRelatedHTTPCode();
+
+    /**
+     * Return the name of the Java class name to use to recreate the exception.
+     *
+     * @return String name of the fully-qualified java class name
+     */
+    String getExceptionClassName();
+
+    /**
+     * Return the name of any nested exception that may indicate the root cause of the exception.
+     *
+     * @return exception class name
+     */
+    String getExceptionCausedBy();
+
+    /**
+     * Return the description of the activity in progress when the exception occurred.
```

Comment:
the comments say activity but the method name is Action. I suggest using the same noun to prevent potential confusion

Reply:
Yes, okay I too think so